src/Umbraco.Web.UI.Client/src/views/common/overlays/treepicker/treepicker.controller.js

```diff
@@ -9,7 +9,9 @@ angular.module("umbraco").controller("Umbraco.Overlays.TreePickerController",
         $scope.section = dialogOptions.section;
         $scope.treeAlias = dialogOptions.treeAlias;
         $scope.multiPicker = dialogOptions.multiPicker;
-        $scope.hideHeader = true;
+        $scope.hideHeader = false;

```

Comment:
@warrenbuckley haha excellent catch 👍 I used it for some debugging and forgot to change it back. Will revert that.